packages/backend-test-utils/src/database/types.ts

```diff
@@ -24,6 +24,8 @@ import { Knex } from 'knex';
  */
 export type TestDatabaseId =
   | 'POSTGRES_13'
+  | 'POSTGRES_12'
+  | 'POSTGRES_11'
   | 'POSTGRES_9'
```

Comment:
Slightly related to testing with more databases, when do we drop PG9? I mean we are fairly strict with typescript/node versions and it would be nice to be in a similar situation with databases. Took a brief look at https://www.postgresql.org/support/versioning/ and it looks like 9 is no longer supported.